src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2486,7 +2489,7 @@ private SSTableMultiWriter writeMemtableRanges(Supplier<Collection<Range<Partiti
                                                              0,
                                                              repairSessionID,
                                                              false,
-                                                             0,
+                                                             commitLogIntervals.build(),
```

Comment:
This part is different, and it doesn't seam like a part of "removing invalid shared collector in flushing". Previously the writer was created with a collector that had empty intervals. Now the collector is built with non-empty intervals. How does this affect the callers (streaming, repair)?

Reply:
It should not have any effect, because the receiver is not the same node (and thus the intervals must be ignored by it). From the point of view of minimizing surprises in the future, it's a bit cleaner to have them.